test/unit/test_run.c

```diff
@@ -0,0 +1,48 @@
+// SPDX-FileCopyrightText: 2021 Anton Kochkov <anton.kochkov@gmail.com
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_socket.h>
+#include "minunit.h"
+
+static const char *runprofile =
+	"program=/bin/ls\n"
```

Comment:
instead of running /bin/ls only on unix you can have a simple program like what i did with `test/unit/auxilliary`.

Reply:
Good idea, done just that. Lets see if it work on Windows...